crates/iceberg-ext/src/configs/namespace.rs

```diff
@@ -0,0 +1,129 @@
+use super::{ConfigParseError, ConfigProperty, Location, NotCustomProp, ParseFromStr};
+use std::collections::HashMap;
+use std::fmt::Debug;
+
+#[derive(Debug, PartialEq, Default)]
+#[allow(clippy::module_name_repetitions)]
```

Comment:
let's maybe put a `#![allow(clippy::module_name_repetitions)]` on `lib.rs` or at least `mod.rs`, I don't think this is a helpful lint, we have a lot of places where we allow it by now

Reply:
We already have it on lib level - just old habit